x/ethbridge/legacy/v42/migrate.go

```diff
@@ -6,8 +6,8 @@ import (
 )
 
 func Migrate(state v039ethbridge.GenesisState) *types.GenesisState {
```

Comment:
We need a migration that will take cethReceiver from the old system and call SetNativeToken for the new system

Reply:
it is done in the ethbridge module's genesis.go